maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -276,14 +277,14 @@ object MaestroSessionManager {
         val xcTestInstaller = LocalXCTestInstaller(
             logger = IOSDriverLogger(LocalXCTestInstaller::class.java),
             deviceId = deviceId,
-            host = defaultHost,
+            host = defaultXctestHost,
```

Comment:
How are ipv4 and ipv6 mapped to a host in /etc/hosts?

Can a host with ipv4 communicate to ipv6 as well and vice versa?

Reply:
this is basically the default mapping:

```
127.0.0.1       localhost
::1             localhost
```

XCTest server only listens on IPv6

> Can a host with ipv4 communicate to ipv6 as well and vice versa?

with correct networking setup yes, with the library we're using (FlyingFox) it is only able to listen on ::1 for loopback addresses